packages/playwright-core/src/server/instrumentation.ts

```diff
@@ -27,7 +28,7 @@ export type Attribution = {
   isInternal: boolean,
   browserType?: BrowserType;
   browser?: Browser;
-  context?: BrowserContext;
+  context?: BrowserContext | APIRequestContext;
```

Comment:
This might be a bit surprising to existing code that checks `attribution.context`, won't it?

Reply:
`_captureSnapshot` checks for `sdkObject.attribution.page` and `onAfterCall` checks for `sdkObject.attribution.context`, I believe we are good.